index.html

```diff
@@ -0,0 +1,176 @@
+  <section class="about-bio">
+    <div class="introduction">
+      <h1>I’m Lucky<br> Glad to see<br> you!</h1>
+      <p>I’m a software developer! I can help you<br> build a product, feature or website Look<br> through some of my work and<br> experience! If you like what you see and<br> have a project you need coded, don’t<br> hestiate to contact me.</p>
+    </div>
+         <a class="link-up" href="#">LET'S CONNECT</a>
+            <ul class="social-media-icons">
+                <li><a href="#"><i class="fab fa-twitter"></i></a></li>
+                <li><a href="www.linkedin.com/in/lucky-james-osuagwu-a8677b171"><i class="fab fa-linkedin-in"></i></a></li>
+                <li><a href="#"><i class="fab fa-medium-m"></i></a></li>
+                <li><a href="https://github.com/Goodluck5956"><i class="fab fa-github"></i></a></li>
+                <li><a href="#"><i class="fab fa-angellist"></i></a></li>
+            </ul>
+    </section>
+
+        <main class="grid">
+          <section class="work-one work">
+            <img src="./templates/SnapshootPortfolio(6).png" alt="snap1">
+            <h2>Tonic</h2>
+            <ul class="work-history">
+              <li class="cover">CANOPY</li>
+                <li><img class="counter" src="./templates/Counter(1).svg" alt="Counter"></li>
+                  <li class="career">Back End Dev</li>
+                <li><img class="counter" src="./templates/Counter(1).svg" alt="Counter"></li>
+              <li class="career">2015</li>
+            </ul>
+            <p class="body-detail">
+              A daily selection of privately<br> personalized reads; no accounts or<br> sign-ups required.
+            </p>
+            <ul class="courses">
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+          <form action="#">
+              <button class="btn-body" type="submit" name="submit">see project</button>
+          </form>
+          </section>
+
+          <section class="work-one work">
+            <img class ="img-work" src="./templates/snap2(1).svg" alt="Tonic">
+            <h2>Multi-Post<br>Stories</h2>
+            <ul class="work-history">
+              <li class="cover">CANOPY</li>
+              <li><img class="counter" src="./templates/Counter(1).svg" alt="counter"></li>
+              <li class="career">back end dev</li>
+              <li><img class="counter" src="./templates/Counter(1).svg" alt="counter"></li>
+              <li class="career">2015</li>
+            </ul>
+            <p class="body-detail">
+              A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+              <ul class="courses">
+                <li>html</li>
+                <li>css</li>
+                <li>javaScript</li>
+              </ul>
+            <form action="#">
+                <button class="btn-body" type="submit" name="submit">see project</button>
+            </form>
```

Comment:
- [ ] Since your not submitting any form. I think it's better to remove the form tag and only use the button.
- [ ] Kindly add the button styling: Active, Hover, and Pressed. For the buttons as described in the original design.
![image](https://user-images.githubusercontent.com/43698511/130152017-0e45d17f-fcfc-4b11-8a6b-ee180fbb1b64.png)